assets/scripts/library-api.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-module.exports = true;
+const getBooks = function(){
+  return $.ajax({
+    url: 'http://localhost:3000/books',
+    method: 'GET',
+  });
+};
+
+const getInputBook = function(bookIndex){
```

Comment:
`getInputBook` isn't a great name. Can you think of a better one for what this function does?